nevergrad/benchmark/experiments.py

```diff
@@ -263,6 +263,56 @@ def spsa_benchmark(seed: Optional[int] = None) -> Iterator[Experiment]:
                     yield Experiment(function, optim, budget=budget, seed=next(seedg))
 
 
+@registry.register
+def realworld(seed: Optional[int] = None) -> Iterator[Experiment]:
+    # This experiment contains:
+    # - a subset of MLDA (excluding the perceptron)
+    # - ARCoating https://arxiv.org/abs/1904.02907
+    # - The 007 game
+    
+    # MLDA stuff, except the Perceptron.
+    funcs: List[Union[InstrumentedFunction, rl.agents.TorchAgentFunction]] = [
+        _mlda.Clustering.from_mlda(name, num, rescale) for name, num in [("Ruspini", 5), ("German towns", 10)] for rescale in [True, False]
+    ]
+    funcs += [
+        _mlda.SammonMapping.from_mlda("Virus", rescale=False),
+        _mlda.SammonMapping.from_mlda("Virus", rescale=True),
+        _mlda.SammonMapping.from_mlda("Employees"),
+    ]
+    # funcs += [_mlda.Perceptron.from_mlda(name) for name in ["quadratic", "sine", "abs", "heaviside"]]
+    funcs += [_mlda.Landscape(transform) for transform in [None, "square", "gaussian"]]
+
+    # Adding ARCoating.
+    funcs += [ARCoating()]
+
+    # 007 with 100 repetitions, both mono and multi architectures.
+    base_env = rl.envs.DoubleOSeven(verbose=False)
```

Comment:
@jrapin    will DoubleOSeven as instantiated here be a deterministic optimization problem ?

Reply:
(for this xp, not centered on noise, I'd like to get rid of noise issues at least for the moment --- a second version of "realworld" should be stochastic, with different problems)

Reply:
It will be noisy:
- random agent is not deterministic
- the other agents are set to `deterministic=False`

The noise should be slightly smoothed out by `num_repetitions=100`, but that's still noisy I would think

Reply:
At some point I'll probably update this part, because it's a bit complicated so there's probably a better way to configure this... I have no idea how though :s